standard/documentation-comments.md

```diff
@@ -616,19 +650,22 @@ enum Color { Red, Blue, Green }
 
 namespace Acme
 {
-   interface IProcess { ... }
-   struct ValueType { ... }
-   class Widget: IProcess
-   {
-      public class NestedClass { ... }
-      public interface IMenuItem { ... }
-      public delegate void Del(int i);
-      public enum Direction { North, South, East, West }
-   }
-   class MyList<T>
-   {
-      class Helper<U,V>{ ... }
-   }
+    interface IProcess { ... }
+
+    struct ValueType { ... }
+
+    class Widget: IProcess
```

Comment:
I've been adding a space before the colon for inheritance/implementation. Are you happy for us to make that the convention?

Reply:
Yes. I'll make that change.